plugins/optimization-detective/tests/class-optimization-detective-test-helpers.php

```diff
@@ -23,15 +23,17 @@ trait Optimization_Detective_Test_Helpers {
 	 * @phpstan-param ElementDataSubset[] $elements
 	 * @param array[] $elements Element data.
 	 * @param bool    $complete Whether to fully populate the groups.
-	 * @throws Exception But it won't.
+	 * @return positive-int Post ID.
```

Comment:
Related to my comment above: While this technically is a post ID, I think focusing on "URL metrics ID" is more clear on what it is.
```suggestion
	 * @return positive-int URL metrics ID.
```

Reply:
I think better:

```suggestion
	 * @return positive-int ID for od_url_metrics post.
```